Readme.md

```diff
@@ -237,6 +241,14 @@ Set the quickstart to not include any oh-my-zsh plugins from the standard plugin
 
 Sets the quickstart to include the oh-my-zsh plugins from the standard plugin list.
 
+##### zqs enable-ssh-askpass-require
+
+Enable the quickstart to prompt for your ssh passphrase on the command line.
+
+##### zqs diasble-ssh-askpass-require
```

Comment:
s/diasble/disable/

Reply:
That's my bad touch typing.

Reply:
It's always easier for someone else to spot them.